HtmlTextView/src/main/java/org/sufficientlysecure/htmltextview/HtmlHttpImageGetter.java

```diff
@@ -125,10 +125,13 @@ protected void onPostExecute(Drawable result) {
             if (imageGetter == null) {
                 return;
             }
+
+            // re-set text to fix images overlapping text
+            CharSequence text = imageGetter.container.getText();
+            imageGetter.container.setText(text);
+
             // redraw the image by invalidating the container
             imageGetter.container.invalidate();
-            // re-set text to fix images overlapping text
-            imageGetter.container.setText(imageGetter.container.getText());
```

Comment:
Can you explain what this change does? I was under the impression that the order of these method calls matter.